source/reference/keyboard_shortcuts_osx.rst

```diff
@@ -61,6 +61,8 @@ Editing
 +-----------------+-----------------------------------------------------------+
 | ⌘ + ⇧ + V       | Paste and indent correctly                                |
 +-----------------+-----------------------------------------------------------+
+| ⌘ + K, ⌘ + V    | Paste selecting item from paste history                   |
```

Comment:
It shows you a list of your previous clipboard items. You can "select" one of them and then trigger the paste action. Feel free to make this clearer since English is not my first language.